pkg/apis/vpc/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -44,6 +44,12 @@ type VPCNetworkConfigurationSpec struct {
 type SharedSubnet struct {
 	// NSX path of Subnets created outside of the Supervisor to be associated with this vSphere Namespace
 	Path string `json:"path"`
+	// Indicates if this subnet is the default subnet for pods.
```

Comment:
Indicates if this subnet is used as Pod default network? since not only 1 subnet can be used for Pod default.

Reply:
Done